api/migrations/0001_initial.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 4.1.4 on 2022-12-12 21:47
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
I don't think this is how you want to implement the board modeling, because you don't have the necessity of storing in a DB every board state , we need to talk how we want to implement this.

Reply:
that's true, it doesn't make sense, I was doing a Django REST api tutorial so I was not really thinking if it  was logical implement that. I would correct that.